cypress/integration/basic.spec.ts

```diff
@@ -1,8 +1,54 @@
 /// <reference types="cypress" />
 
-describe('Smoke Test', () => {
-  it('renders fallback component', () => {
-    cy.visit('http://localhost:1234')
+describe('Wrapping Component', () => {
+  it('properly renders home page', () => {
+    cy.visit('http://localhost:1234/')
```

Comment:
Is this rather a `before` step or so if all the other cases are dependent on it? e.g. `.only` scoping on a test below would fail, right?